src/lib.rs

```diff
@@ -420,4 +479,59 @@ mod tests {
             deposit + deposit + 1 - 2 * ACCESS_KEY_ALLOWANCE
         );
     }
+
+    #[test]
+    fn test_create_advanced_account() {
+        // Create a new instance of the linkdrop contract
+        let mut contract = LinkDrop::new();
+        // Create the public key to be used in the test
+        let pk: PublicKey = "qSq3LoufLvTCTNGC3LJePMDGrok8dHMQ5A1YD9psbiz"
+            .parse()
+            .unwrap();
+        // Default the deposit to an extremely small amount
+        let deposit = 1_000_000;
+
+        // Create options for the advanced account creation
+        let options: CreateAccountOptions = CreateAccountOptions {
+            full_access_keys: Some(vec![pk.clone()]),
+            limited_access_keys: Some(vec![LimitedAccessKey {
+                public_key: pk.clone(),
+                allowance: U128(100),
+                receiver_id: linkdrop(),
+                method_names: "send".to_string(),
```

Comment:
I suggest test two full access keys and two limited access keys, which covers one and multiple key case.